packages/gestalt/src/Badge.js

```diff
@@ -10,7 +10,7 @@ type Props = {|
   text: string,
 |};
 
-export default function Badge(props: Props) {
+export default function Badge(props: Props): React.Element<'span'> {
```

Comment:
Does it make sense to be this specific? I typically type all component returns as `React.Node` since it shouldn't matter externally what it is.

Reply:
@btraut those were automatically added since flow inferred the type as `React.Element<'span'>`. Still changed them all to `React.Node` since I agree externally we should not expose that information.